TINY-8175: Changed isOSX to isMacOS (#7411)
tinymce/tinymce: modules/tinymce/CHANGELOG.md

### Changed
- The `editor.getContent()` API can provide custom content by preventing and overriding `content` in the `BeforeGetContent` event. This makes it consistent with the `editor.selection.getContent()` API #TINY-8018

> - The `editor.env.isOSX` API function changed to `editor.env.isMacOS` #TINY-8175

Reviewer: ```suggestion
- The `tinymce.Env.os.isOSX` API has been renamed to `tinymce.Env.os.isMacOS` #TINY-8175
```
`editor.env` doesn't exist so this needs to use the correct API names.
